cloud_experts_tutorials/cloud-experts-rosa-with-hcp-private-offer-acceptance-and-sharing.adoc

```diff
@@ -0,0 +1,187 @@
+====
+
+. If all the details match your expectations, click the *Create contract* button, shown in Figure 5, for private offers accepted by an AWS account that does not have the {hcp-title} activated yet and is creating the first contract for this service.
++
+.Create contract button
++
+image::rosa-create-contract-button.png[]
+
+. Or click the *Upgrade current contract button* for agreement-based offers as seen in Figures 4 and 6.
++
+.Upgrade contract button
++
+image::rosa-upgrade-contract-button.png[]
+
+. There is one more confirmation modal window shown, similar to the one shown in Figure 7.
++
+.Private offer acceptance confirmation window
++
+image::rosa-private-offer-acceptance-confirmation-window.png[]
+
+. If the accepted private offer service start date is set to be immediately following the offer acceptance, you can click the *Set up your account* button in the confirmation modal window shown in Figure 8.
++
+.Subscription confirmation
++
+image::rosa-subscription-contfirmation.png[]
+
+. If the accepted private offer has a future start date specified, return to the private offer page after the service start date, these are always in UTC time zone, and click the *Setup your account* button to proceed with the Red{nbsp}Hat and AWS account linking. With no agreement active, the account linking described below is not triggered, the _Account setup_ process can be done only after the _Service start date_.
+
+== Sharing a private offer
+
+. Clicking the *Set up your account* button in the previous step takes you to the AWS and Red{nbsp}Hat account linking step. At this time, you are already logged in with the AWS account that accepted the offer. If you are not logged in with a Red{nbsp}Hat account, you will be prompted to do so.
++
+{hcp-title} entitlement is shared with other team members through your Red{nbsp}Hat organization account. All existing users in the same Red{nbsp}Hat organization are able to select the billing AWS account that accepted the private offer by following the above described steps. You can link:https://www.redhat.com/wapps/ugc/protected/usermgt/userList.html[manage users in your Red{nbsp}Hat organization], when logged in as the Red{nbsp}Hat organization administrator, and invite or create new users.
++
+[NOTE]
+====
+{hcp-title} private offer cannot be shared with AWS linked accounts through the AWS License Manager.
+====
+
+. Check link:https://access.redhat.com/customer-service-users[this user management FAQ] for more details about Red{nbsp}Hat account user management tasks. Make sure that the already logged in Red{nbsp}Hat account includes all users that are meant to be ROSA cluster deployers benefiting from the accepted private offer.
```

Comment:
Step 2 doesn't really feel like a step. Maybe we can say 
"Add any users that you want to deploy ROSA clusters. Check [this user management FAQ](https://access.redhat.com/customer-service-users) for more details about Red Hat account user management tasks."